File: PHP/automorphic.php
<?php
// working...success
function check_automorphic(int $n):int{
    $copy=$n;
    $sq=($n*$n);
    while(floor($copy)){
        if (($copy%10)!=($sq%10)){return 0;}
        $copy=$copy/10;
        $sq=$sq/10;
    }
    return 1;
}
echo "enter a number to check for automorphic number\n";
$user=(int)readline();
$var=((check_automorphic($user)==1)?"TRUE":"FALSE");
echo ">>".$var;
?>

File: java/automorphic.java
import java.util.*;
public class automorphic{
	// working...success...
	public static Boolean is_automorphic(int n){
			int num=n,sq=n*n;
			while(num>0){
				if((num%10) != (sq%10)){return false;}
				num/=10;
				sq/=10;
			}
			return true;
		}
	public static void main(String dd[]){
		Scanner nrt=new Scanner(System.in);
		System.out.println("enter a number");
		int user=nrt.nextInt();
		automorphic A=new automorphic();
		System.out.println((A.is_automorphic(user)?"Automorphic":"Not Automorphic"));
	}
}